core/src/main/scala/org/apache/spark/memory/StaticMemoryManager.scala

```diff
@@ -104,6 +104,8 @@ private[spark] class StaticMemoryManager(
 
 private[spark] object StaticMemoryManager {
 
+  private val MIN_MEMORY_BYTES = 450 * 1024 * 1024
```

Comment:
Yeah I just mean let's literally refer to the same constant rather than copying it. Maybe it needs to live somewhere else to make sense to access in both classes.